cypress/integration/tiles.js

```diff
@@ -0,0 +1,286 @@
+describe('Default Tiles functionality', () => {
```

Comment:
There is no "default" tile. This is the "Text tile".

Reply:
maybe we can review the structure of the tests later, for something that makes more sense.
right now I just put all tests  in the same `describe`